src/main/scala/org/broadinstitute/dsde/firecloud/utils/TSVFormatter.scala

```diff
@@ -119,7 +119,7 @@ object TSVFormatter {
     * @return Entity name as first column header, followed by matching entity attribute labels
     */
   def makeEntityHeaders(entityType: String, allHeaders: Seq[String], requestedHeaders: Option[IndexedSeq[String]]): IndexedSeq[String] = {
-    val memberPlural = pluralizeMemberType(memberTypeFromEntityType(entityType))
+    val memberPlural = FlexibleModelSchema.pluralizeEntityType(FlexibleModelSchema.memberTypeFromEntityType(entityType).getOrElse(""))
```

Comment:
Same comment as line 88.

Reply:
this is also used in filtering (filterNot), so i think it's actually ok to default to empty string

Reply:
Good point, though again I think that `getOrElse(entityType)` would be more similar to the previous behavior and feels a little less arbitrary than `""`. However, I haven't fully thought through whether or not this would affect the `filterNot`.